extensions/wrapper/wrapper-api/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/ContractTerminationRequest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Copyright (c) 2024 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ */
+
+package de.sovity.edc.ext.wrapper.api.ui.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@RequiredArgsConstructor
+@Schema(description = "Data for terminating a Contract Agreement")
+public class ContractTerminationRequest {
+
+    @Schema(
+        title = "Termination reason",
+        description = "A short reason why this contract was terminated",
+        maxLength = 64,
```

Comment:
we don't have this for the entire application, and I guess we shouldn't start now, unless there is a special intentful limit on something

Reply:
This was to prevent people from putting the long detail text into this field.
This field must remain brief, and all the legalese must go into the detail section.
Also, we may have people pushing lots of data in this field. I'm already imagining big corporations `base64`ing their PDFs to upload as a details... :(

Reply:
feel free to do any validation to a truly unreasonable number, but please do that in code for now. this place is of no measurable consequence as of now

we also don't document these things via OpenAPI for now, unless we know it has a very specific format, e.g. some special part number, where we know it needs to have exactly 12 digits